tests/monitoring/vm_monitoring.go

```diff
@@ -321,7 +365,7 @@ var _ = Describe("[sig-monitoring]VM Monitoring", Serial, decorators.SigMonitori
 		It("should have kubevirt_vmi_last_api_connection_timestamp_seconds correctly configured", func() {
 			By("Starting a VirtualMachineInstance")
 			vmi := libvmifact.NewAlpine()
```

Comment:
maybe we can use here and in line 381 libvmifact.NewAlpine(libvmi.WithName("testvm-connection-timestamp"))

Reply:
I don't think it's fully random, that's (I think) why we have an ancient timestamp for that namespace/ns combo https://github.com/kubevirt/kubevirt/pull/14383#issuecomment-2786667437

Reply:
> I don't think it's fully random

Can you explain what you mean by that?
https://github.com/kubevirt/kubevirt/blob/91374ce3c07747b89aa5fbe0b53a66c1a0a0b0ca/pkg/libvmi/vmi.go#L52-L55

Reply:
Thought we seed it with the same value but apparently not https://github.com/kubevirt/kubevirt/blob/91374ce3c07747b89aa5fbe0b53a66c1a0a0b0ca/tests/testsuite/fixture.go#L125-L126